webapp/src/Components/Shared/Selects/PlaylistSelect.tsx

```diff
@@ -20,6 +20,7 @@ interface PlaylistMetadata {
     current: boolean
     ranked: boolean
     standardMode: boolean
+    publicMode: boolean
```

Comment:
Do you know of a better place to put this?

Reply:
Its another filtering option IMO.